components/molecules/GameCard/GameCard.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { Fragment, useMemo } from 'react'
+import { View, Image, StyleSheet, Pressable } from 'react-native'
+import Title from '../../atoms/Title/Title'
+import Label from '../../atoms/Label/Label'
+import Text from '../../atoms/Text/Text'
+import useTheme from '../../hooks/useTheme'
+import { Game } from '../../types/Model'
+import LikeStat from '../LikeStat/LikeStat'
+
+export interface GameCardProps {
+  /** Game's thumbnail url
+   *
+   * Example: https://domain.com/images/image.png
+   */
+  imageUrl: string
+  /** Game's name/title
+   *
+   * Example: PHILIPS Tower Defense Simulator
+   */
+  name: string
+  /** Number of likes by friends
+   *
+   * Example: 10
+   */
+  likesByFriends?: number
+  /** Likes on the platform:
+   *
+   * Example: 7200000
+   */
+  likesOnPlatform?: number
+  /** New Notifications Count:
+   *
+   * Example: 5
+   * Default: 0
+   */
+  notificationsCount?: number
+  /** Subtitle
+   *
+   * Used in mini version
+   */
+  callout?: string
+  /** on Tap/Click/Press function */
+  onPress: (game: Game) => void
```

Comment:
@nbonatsakis what do you think about this definition for the onPress prop? What should its method signature be? I would have assumed `() => void` but I see Rehan has defined a Game interface (see the import within this file) and then when onPress is handled a Game object gets constructed and passed back into this onPress handler. Is that a good idea for our use cases? We'll obviously be declaring a lot of onPress props within this component library.

Reply:
I would've also expected that `() => void` signature. I can't think of a situation where we wouldn't have the input object in scope when handling the onPress.

Reply:
@J-Rehan What was your thinking around this? All of our components just use `() => void` and we've never had a challenge with that. As Nick mentioned, anytime we're declaring the method we have all the context we need at the time of instantiating the component.

I think you can change this onPress handler and delete the whole Game interface.

Reply:
Oh, nevermind, you already made this change. :)